### ISSUE TITLE
GQL Fields with Underscores producing incorrectly in 2021.1.23

### ISSUE THREAD

**Issue by djchapm** [manifold-systems/manifold#303]
**Describe the bug**
Consider GQL:
```
input LONGQueryArgument {
  _eq: Long
  _ne: Long
  _ge: Long
  _le: Long
  _lt: Long
  _gt: Long
  _in: [Long]
  _has: Boolean
}

input my_filter {
  AND: [my_filter]
  OR: [my_filter]
  orderId: LONGQueryArgument
}
```
Prior to 2021.1.23, generated graphql (converted to JSON here) might look like:
```
      "AND": [
        {
          "orderId": {
            "_eq": 22
          },
```
Now however, the output is:
```
      "AND": [
        {
          "orderId": {
            "eq": 22
          },
```
**"eq"** instead of **"_eq"**

**To Reproduce**
1. Validate generated graphql for fields containing leading underscore as part of their name.

**Expected behavior**
Underscore retained.

**Desktop (please complete the following information):**
- OS Type & Version: MacOS BigSur
- Java/JDK version: openjdk 11
- IDE version (IntelliJ IDEA or Android Studio): Intellij
- Manifold version: 2021.1.23
- Manifold IntelliJ plugin version: 2021.1.18

**Additional context**

**Stack trace**
Please include a stack trace if applicable

**Comment by rsmckinney** [comment]
Will fix this ASAP. Thanks for reporting!

**Comment by rsmckinney** [comment]
fixed in release 2021.1.24

**Comment by djchapm** [comment]
Hey - the underscores are fixed but I'm having additional issues... let me know if you want me to move to a new issue.... 
1. declare builder:
private final schema.MyMessage_filter.Builder myMessageBuilder = schema.MyMessage_filter.builder();

If I break in the debugger now - the class shows as:
(**DataBindings**@2221) for the class type... 

The issue is that I have a method:
**setField(String fieldName, GqlBuilder builder)**

Things blow up as soon as I try to make the call:
**setField ("whatever", myMessageBuilder);**

Exception:
**java.lang.ArrayStoreException: manifold.json.rt.api.DataBindings**

the top of stack trace points to the line where I try to call setField above.

**Comment by rsmckinney** [comment]
Are you using latest manifold release (2021.1.24)?  I ran that same code without any issue:
```java
  MovieQuery.Builder b = MovieQuery.builder();
  // --> b is a proxy instance here
  MovieQuery q = buildit(b);
  
  <V extends GqlType> V buildit(GqlBuilder<V> builder) {
    return builder.build();
  }

```

**Comment by djchapm** [comment]
Yea on latest.... looks like this is the issue - when I construct my builder, I'm acutallly setting one of the default fields at the same type... so my declaration is actually:

`private final schema.MyMessage_filter.Builder myMessageBuilder = schema.MyMessage_filter.builder().withAccountType(CHANGE);
`
so looks like the 'with' methods are not returning a type that extends GqlBuilder anymore?
Things work if I change it from the above to:

`
private final schema.MyMessage_filter.Builder myMessageBuilder = schema.MyMessage_filter.builder();
...
myMessageBuilder.withAccountType(CHANGE);
`

Debugger in this case shows the builder as ($ManProxy7) instead of (DataBindings@...)

**Comment by rsmckinney** [comment]
Gotcha.  I’ll have a fix ready by eod.

**Comment by rsmckinney** [comment]
Fixed in release 2021.1.25, see #304
